resources/charlie/chainspec.toml

```diff
@@ -43,9 +43,12 @@ booking_duration_millis = 0
 entropy_duration_millis = 0
 # Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
 # 172800000 ms = 2 days.
-voting_period_duration_millis = 0
-# Integer between 0 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
-finality_threshold_percent = 20
+voting_period_duration_millis = 172800000
```

Comment:
I think this is currently ignored ([NDRS-364](https://casperlabs.atlassian.net/browse/NDRS-364)), and determined by the `AUCTION_DELAY` contant.

Reply:
Just updated to match with resolve conflict with master.